### PR TITLE
Website ryland

### PR DESCRIPTION
**Pull request by rmsepic**
There are still a few more things to do. The image drop page needs to be finished and the table on the front page needs to be populated. I left Yoosuf's image drop for now because we know it works, however, it is was not working for me, which may be due to some of the things I had to comment out in order for the program to run.

This is mainly a design prototype. If there are aspects, features, or edits anyone would think is a good idea just let me know. 

You can check out my README under the directory 'website', but the code will be run under 'PP_APP'. All that needs to be done to run this is to download everything from requirements.txt into a virtual environment and then run the command "python manage.py runserver".

### CODE REVIEW

File: PP_APP/server/frontend/templates/frontend/about.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+      <p class="para p-4"> Our engineering design showcase youtube video will go here, but for the meantime
+      	enjoy Yoosuf.
+      </p>
+    </div>
+  </div>
+
+  <div class="container p-5 video-cont">
+    <div class="embed-responsive embed-responsive-16by9">
+        <iframe src="https://www.youtube.com/embed/PUzpwOkY0h8" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
[the remark this answers is not in the document]
Comment (rmsepic) [reply PP_APP/server/frontend/templates/frontend/about.html:36]: That's a good catch. I should've watched it more carefully.

File: PP_APP/server/frontend/templates/frontend/home.html [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+											</form>
+										</div>
+									</div>
+								</th>
+							</thead>
+
+							<tbody>
+								<tr>
+									<th scope="row">#1</th>
+									<td><img src="website_images/target_logo.svg" class="logo-img"></td>
```
[the remark this answers is not in the document]
Comment (rmsepic) [reply PP_APP/server/frontend/templates/frontend/home.html:84]: Yeah, the images are in a different directory now, the links will need to be changed. For designing the frontend should the company get its image from an image folder full of company logos or should we store an image with the company class in the database?

[the remark this answers is not in the document]
Comment (fstamenkovic) [reply PP_APP/server/frontend/templates/frontend/home.html:84]: Good questions. I'd go with the database, but I have less knowledge about this part. @ShafiTek should pitch in

File: PP_APP/server/frontend/README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+## Styling
+
+*main.css*: is a compiled *.css* file containing all of the bootstrap classes with 
+some modifications that I added. The modifications can be found in *main.scss*.
+
+*style.css*: contains classes I created that were **NOT** based on *Bootstrap*.
+
+*images*: is a directory under *static/css* which contains all the images needed 
+to run the website.
+
```
Comment (fstamenkovic) [inline PP_APP/server/frontend/README.md:34]: Thank you.

File: PP_APP/README.md [hunk 4]
```diff
@@ -1,5 +1,8 @@
-# HOW TO SETUP AND RUN THE API FOR THE FIRST TIME
+[Yoosuf][#yoosuf]
+
+<a name="yoosuf"/>
```
Comment (fstamenkovic) [inline PP_APP/README.md:3]: Not sure what this is.

File: PP_APP/server/frontend/templates/frontend/about.html [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+        <div class="col-sm offset-2">
+          <h3>Filip Stamenkovic</h3>
+          <a href="https://github.com/fstamenkovic">Github</a> |
+          <a href="https://www.linkedin.com/in/fstamenk/">LinkedIn</a>
+        </div>
+      </div>
+    </div>
+  </div> <!--- Container for background -->
+
+{% endblock content %}
```
Comment (fstamenkovic) [inline PP_APP/server/frontend/templates/frontend/about.html:70]: Looks fantastic!

File: PP_APP/server/frontend/templates/frontend/home.html [hunk 6]
```diff
@@ -0,0 +1,131 @@
+{% extends "frontend/base.html" %}
+{% load static %}
+{% block content %}
+		<div class="container-fluid p-3" style="background-image: url('../../static/css/images/background/green_background.jpg')">
+			<div class="container">
+				<div class="row py-lg-5">
+					<div class="col-6">
+						<b style="font-size: 60px">Using computer vision to better the environment</b>
```
Comment (fstamenkovic) [inline PP_APP/server/frontend/templates/frontend/home.html:8]: We'll change this at some other opportunity.

File: PP_APP/server/frontend/templates/frontend/success.html [hunk 7]
```diff
@@ -0,0 +1,41 @@
+{% extends "frontend/base.html" %}
```
Comment (fstamenkovic) [inline PP_APP/server/frontend/templates/frontend/success.html:1]: Good enough for now. We'll update this page a bit when we start hooking it up to the backend. Really depends on what kind of API from Django we settle on.

File: PP_APP/server/frontend/templates/frontend/upload.html [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+      </div>
+  </div>
+
+  <div class="row">
+    <div class="col justify-content-center">
+      <button class="btn btn-primary btn-lg" onclick="success()">UPLOAD</button>
+    </div>
+  </div>
+</div>
+
```
Comment (fstamenkovic) [inline PP_APP/server/frontend/templates/frontend/upload.html:41]: We'll get rid of the double fields when we start hooking this up to the backend. Not difficult.

**Pull request merged**